sotodlib/preprocess/processes.py

```diff
@@ -14,6 +14,8 @@
 from .pcore import _Preprocess, _FracFlaggedMixIn
 from .. import flag_utils
 
+import so3g
```

Comment:
Instead of a blanket import please import just the modules you need `from so3g.proj import RangesMatrix`